frontend/invites/newInviteController.js

```diff
@@ -118,7 +122,7 @@
             return instObj.getFullAddress();
         };
 
-        function deleteInvite() {
+        function deleteInviteAndReloadUser() {
```

Comment:
Use "and" in the name of a function means that you are doing too many things in it.